mava/systems/ppo/types.py

```diff
@@ -52,6 +52,7 @@ class LearnerState(NamedTuple):
     key: chex.PRNGKey
     env_state: State
     timestep: TimeStep
+    dones: Done
```

Comment:
Because you added dones to the type here it caused some issues in Sebulba ippo type checker 
`mava/systems/ppo/sebulba/ff_ippo.py:215: error: Too many values to unpack (5 expected, 6 provided)`
and this is because sebulba using same type, so maybe make other type `LearnerStateSebulba` until we make a PR for gae related to sebulba systems